scipy/stats/_multivariate.py

```diff
@@ -572,30 +567,25 @@ def __call__(self, mean=None, cov=1, allow_singular=False, seed=None,
                                           allow_singular=allow_singular,
                                           seed=seed, inverse_cov=inverse_cov)
 
-    def _process_parameters(self, dim, mean, cov, inverse_cov):
```

Comment:
Drive-by refactoring: callers always passed `dim=None` to this method so I simply removed it.